eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl

```diff
@@ -79,7 +85,15 @@ SOFTWARE.
         </xsl:otherwise>
       </xsl:choose>
     </xsl:variable>
-    <xsl:value-of select="concat($p, eo:clean($c))"/>
+    <xsl:variable name="pre" select="concat($p, eo:clean($c))"/>
+    <xsl:choose>
+      <xsl:when test="string-length($pre)&gt;250">
+        <xsl:value-of select="concat(substring($pre, 1, 25), $alt)"/>
```

Comment:
@mximp what about possible conflicts between two long class names? If you just cut them, you will have them conflicted for sure.

Reply:
@yegor256 that's why I pass `$alt` which is combination of line & position. Within a file it should be unique postfix.